lib/VeresOne.js

```diff
@@ -101,8 +101,8 @@ class VeresOne {
     }
 
     // get private key
-    const invokeKeyNode = didDocument.suiteKeyNode({
-      suiteId: 'capabilityInvocation'
+    const invokeKeyNode = didDocument.proofPurposeKey({
```

Comment:
This is an odd function name ... what is it doing? It's getting something, right? Or is it creating something... or some combination of both?

Reply:
Also, this wouldn't just be for keys, right? But for any verification method?